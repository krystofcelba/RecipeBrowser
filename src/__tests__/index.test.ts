import { Navigation } from 'react-native-navigation';

jest.mock('../redux', () => () => {});
jest.mock('../screens', () => ({ registerScreens: () => {} }));

import App from '..';

describe('App', () => {
  it('should start single screen app as expected', () => {
    Navigation.startSingleScreenApp = jest.fn();
    new App({});
    expect(Navigation.startSingleScreenApp.mock.calls).toMatchSnapshot();
  });
});
