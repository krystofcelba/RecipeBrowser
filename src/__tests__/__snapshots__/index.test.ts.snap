// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should start single screen app as expected 1`] = `
Array [
  Array [
    Object {
      "appStyle": Object {
        "keepStyleAcrossPush": false,
      },
      "screen": Object {
        "navBarBackgroundColor": "#81b640",
        "navBarTextColor": "#ffff",
        "navBarTextFontFamily": "Avenir",
        "screen": "recipeBrowser.RecipesList",
        "statusBarColor": "#508607",
        "statusBarTextColorScheme": "light",
      },
    },
  ],
]
`;
